rllib/BUILD

```diff
@@ -2054,15 +2047,6 @@ py_test(
     args = ["--stop-timesteps=2000", "--run=QMIX"]
 )
 
-py_test(
```

Comment:
Why did this have to go?

Reply:
There were some issues with QMIX + APEX, and I just decided to remove it since we have no test cases or benchmarks showing this is a useful algorithm.

Reply:
Fair enough. Yeah, we should put some time into it to see, whether it actually works.